lib/ci-stack.ts

```diff
@@ -100,18 +100,19 @@ export class CIStack extends Stack {
 
     const useSsl = useSslParameter === 'true';
 
-    const runWithOidcParameter = `${props?.runWithOidc ?? this.node.tryGetContext('runWithOidc')}`;
-    if (runWithOidcParameter !== 'true' && runWithOidcParameter !== 'false') {
-      throw new Error('runWithOidc parameter is required to be set as - true or false');
+    let authType = `${props?.authType ?? this.node.tryGetContext('authType')}`;
+    if (authType.toString() === 'undefined') {
+      authType = 'default';
```

Comment:
What is the default type here?

Reply:
I believe it is the plain old user/pass mechanism that jenkins provides.

Reply:
Whatever is the base_jenkins.yaml. @rishabh6788 is right by default is user/pass as before